verticapy/performance/vertica/qprof.py

```diff
@@ -1662,6 +1662,47 @@ def get_qplan_profile(
                 **style_kwargs,
             )
         return vdf
+    
+    # Step 10: Query events
+    def get_query_events(self) -> vDataFrame:
+        """
+        Returns a vDataFrame that contains a table listing query events
+
+        Parameters
+        ----------
+        None
+
+        Returns
+        -------
+        A vDataframe that contains a table listing query events
+
+        Examples
```

Comment:
Yes, and please, add a more detailed example in the main class example above.

Reply:
I have added a detailed example the query events function that I added.